File: src/main/java/com/learnershi/rclasssocket/repository/ClassRoomStudyDataMapRepository.java
package com.learnershi.rclasssocket.repository;

import com.learnershi.rclasssocket.entity.ClassRoomStudyDataMap;
import org.springframework.data.mongodb.repository.ReactiveMongoRepository;

public interface ClassRoomStudyDataMapRepository extends ReactiveMongoRepository<ClassRoomStudyDataMap, String> {
}

File: src/main/java/com/learnershi/rclasssocket/repository/ClassRoomStudyDataMapRepository.kt
package com.learnershi.rclasssocket.repository

import com.learnershi.rclasssocket.entity.ClassRoomStudyDataMap
import org.springframework.data.mongodb.repository.ReactiveMongoRepository

interface ClassRoomStudyDataMapRepository : ReactiveMongoRepository<ClassRoomStudyDataMap?, String?>